site/guide/basics/strings-and-numbers.md

```diff
@@ -1,32 +1,57 @@
 # Strings and Numbers
 
-The basic building blocks of the language are numbers, strings and expressions
+The basic building blocks of the relevance language are numbers, strings and the expressions
 that combine them.
 
+The "Q:" is the relevance query that is being run and the, "A:" is the answer to our query. We will discover that queries can be complex and the answers to a query can be of any type, but more on that soon!
+
+## Strings and Integers
+
+Strings are a set of characters (a-z,0-9,!@#$) that are surrounded by quotes. To start, we will write a relevance string that outputs, "hello world"
```

Comment:
Should be either "Strings are sets of characters" or "A string is a set of characters".